File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/question/FurtherReadingDocumentTest.java
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;

/**
 * Unit test class for {@link FurtherReadingDocument}
 */
public class FurtherReadingDocumentTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(FurtherReadingDocument.class)
				.verify();
	}
}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/persistence/dynamodb/item/question/FurtherReadingDocumentTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.persistence.dynamodb.item.question

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Unit test class for [FurtherReadingDocument]
 */
class FurtherReadingDocumentTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(FurtherReadingDocument::class.java)
				.verify()
	}
}